src/main/java/org/carlmontrobotics/commands/MoveToNote.java

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package org.carlmontrobotics.commands;
+
+import org.carlmontrobotics.Constants.Limelightc;
+import org.carlmontrobotics.subsystems.Drivetrain;
+import org.carlmontrobotics.subsystems.Limelight;
+import org.carlmontrobotics.subsystems.LimelightHelpers;
+
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class MoveToNote extends Command {
+  private final Drivetrain dt;
+  private final Limelight ll;
+    private Timer timer = new Timer();
+  /** Creates a new MoveToNote. */
+  public MoveToNote(Drivetrain dt, Limelight ll) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(this.dt=dt);
+    addRequirements(this.ll=ll);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    new AlignToNote(dt);
```

Comment:
I don't think you want this. It will *not* cause the `AlignToNote()` command to run if that is what you are trying to achieve.